Outdated dependency `node-hid` fails to build

After a recent update, `node-hid` fails to build for me, which causes `npm install msi-keyboard` to fail with output such as:
```
npm ERR! In file included from ../src/HID.cc:31:
npm ERR! ../../nan/nan.h: In function ‘void Nan::SetAccessor(v8::Local<v8::ObjectTemplate>, v8::Local<v8::String>, Nan::GetterCallback, Nan::SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, Nan::imp::Sig)’:
npm ERR! ../../nan/nan.h:2544:19: warning: ‘void v8::ObjectTemplate::SetAccessor(v8::Local<v8::Name>, v8::AccessorNameGetterCallback, v8::AccessorNameSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::Local<v8::AccessorSignature>, v8::SideEffectType, v8::SideEffectType)’ is deprecated: Do signature check in accessor [-Wdeprecated-declarations]
npm ERR!  2544 |   tpl->SetAccessor(
npm ERR!       |   ~~~~~~~~~~~~~~~~^
npm ERR!  2545 |       name
npm ERR!       |       ~~~~         
npm ERR!  2546 |     , getter_
npm ERR!       |     ~~~~~~~~~      
npm ERR!  2547 |     , setter_
npm ERR!       |     ~~~~~~~~~      
npm ERR!  2548 |     , obj
npm ERR!       |     ~~~~~          
npm ERR!  2549 |     , settings
npm ERR!       |     ~~~~~~~~~~     
npm ERR!  2550 |     , attribute
npm ERR!       |     ~~~~~~~~~~~    
npm ERR!  2551 |     , signature);
npm ERR!       |     ~~~~~~~~~~~~   
npm ERR! In file included from /home/asaparov/.cache/node-gyp/18.0.0/include/node/v8-function.h:15,
npm ERR!                  from /home/asaparov/.cache/node-gyp/18.0.0/include/node/v8.h:33,
npm ERR!                  from /home/asaparov/.cache/node-gyp/18.0.0/include/node/node.h:63,
npm ERR!                  from ../src/HID.cc:30:
npm ERR! /home/asaparov/.cache/node-gyp/18.0.0/include/node/v8-template.h:838:8: note: declared here
npm ERR!   838 |   void SetAccessor(
npm ERR!       |        ^~~~~~~~~~~
npm ERR! ../src/HID.cc: At global scope:
npm ERR! ../src/HID.cc:67:5: error: ISO C++17 does not allow dynamic exception specifications
npm ERR!    67 |     throw(JSException);
npm ERR!       |     ^~~~~
npm ERR! ../src/HID.cc:70:5: error: ISO C++17 does not allow dynamic exception specifications
npm ERR!    70 |     throw(JSException);
npm ERR!       |     ^~~~~
npm ERR! ../src/HID.cc:152:3: error: ISO C++17 does not allow dynamic exception specifications
npm ERR!   152 |   throw(JSException)
npm ERR!       |   ^~~~~
npm ERR! ../src/HID.cc:163:3: error: ISO C++17 does not allow dynamic exception specifications
npm ERR!   163 |   throw(JSException)
npm ERR!       |   ^~~~~
npm ERR! ../src/HID.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE HID::read(Nan::NAN_METHOD_ARGS_TYPE)’:
npm ERR! ../src/HID.cc:256:61: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
npm ERR!   256 |   uv_queue_work(Nan::GetCurrentEventLoop(), req, recvAsync, (uv_after_work_cb)recvAsyncDone);
npm ERR!       |                                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I found that using the version `^2.1.1` of `node-hid` works without changes. So I think this is a simple fix by changing the relevant line in `package.json`?

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   "main": "./index.js",
   "dependencies": {
     "color": "^2.0.0",
-    "node-hid": "^0.5.7"
+    "node-hid": "^0.7.4"
   },
   "devDependencies": {
     "mocha": "*",

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   "main": "./index.js",
   "dependencies": {
     "color": "^3.1.2",
-    "node-hid": "^1.2.0"
+    "node-hid": "^2.1.1"
   },
   "devDependencies": {
     "mocha": "^7.1.1",

